templates/kube-apiserver.yaml

```diff
@@ -29,6 +29,15 @@ spec:
         - --authorization-policy-file=/etc/auth/policies.jsonl
         - --authorization-rbac-super-user=kube-admin
         - --audit-log-path=/dev/stdout
+        - --audit-log-maxsize=50MB
+        - --audit-log-maxbackup=0
+      livenessProbe:
+        httpGet:
+          host: 127.0.0.1
+          port: 8080
+          path: /healthz
+        initialDelaySeconds: 15
+        timeoutSeconds: 15
```

Comment:
I think `periodSeconds` should also be set to 15 along with `timeoutSeconds`
period 10 secs(default) and timeout 15 secs would have overlapping/uneven checks.